atomic_reactor/plugins/pre_check_user_settings.py

```diff
@@ -66,6 +67,16 @@ def appregistry_bundle_label_mutually_exclusive(self):
         ):
             raise ValueError(msg)
 
+    def operator_bundle_from_scratch(self):
+        """Only from scratch image can be used for operator bundle build"""
+        msg = "Operator bundle build can be only 'FROM scratch' build (single stage)"
+        self.log.debug("Running check: %s", msg)
+        if (
+            not self.workflow.builder.base_from_scratch or
+            len(self.workflow.builder.parents_ordered) > 1
```

Comment:
Aren't we missing the conditions to make sure the check would only run for operator bundle builds here?

This would break some builds from non-RPM sources in multistage builds

Reply:
OMG yes :D

Reply:
fixed